src/shared/kilocode/api.ts

```diff
@@ -0,0 +1,2 @@
+// export const getKiloCodeApiUrl = () => "https://kilocode.ai"
+export const getKiloCodeApiUrl = () => "https://app.roocode.com"
```

Comment:
lol 😂  They don't have a way to block us right? I'd just hate for this URL to suddenly stop working for folks if they update their server

Reply:
Nevermind you changed this later on! :D